humbug.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I guesd that calls for support a .dist file in humbug :)
Le 9 mars 2015 19:50, "Rafał Wartalski" <notifications@github.com> a écrit :

> In humbug.json
> <https://github.com/padraic/humbug/pull/122#discussion_r26063551>:
>
> > @@ -0,0 +1,11 @@
> > +{
>
> Note that you don`t share phpunit.xml int your projects. You would rather
> share phpunit.xml.dist.
> For example my humbug.json in humbug often changes its contents. If it is
> in repository there is a chance that anyone can commit by accident with
> invalid contents.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/padraic/humbug/pull/122/files#r26063551>.
>